runhouse/servers/cluster_servlet.py

```diff
@@ -434,11 +412,33 @@ async def astatus(self):
             "system_memory_usage": memory_usage,
             "system_disk_usage": disk_usage,
         }
+
+        # converting status_data to ResourceStatusData instance to verify we constructed the status data correctly
         status_data = ResourceStatusData(**status_data)
-        return status_data
 
-    def status(self):
-        return sync_function(self.astatus)()
+        if send_to_den:
+
+            logger.debug("Sending cluster status to Den")
+            save_in_den_resp = rns_client.save_status_metrics_to_den(
+                status=status_data,
+                cluster_uri=rns_client.format_rns_address(
+                    self.cluster_config.get("name")
+                ),
+                api_server_url=self.cluster_config.get(
+                    "api_server_url", rns_client.api_server_url
+                ),
+            )
+
+            response_info = {
```

Comment:
what if we just returned the response object from the request? that way we could later reference its status code or message pretty conveniently